crypto/asn1/charmap.h

```diff
@@ -2,7 +2,7 @@
  * WARNING: do not edit!
  * Generated by crypto/asn1/charmap.pl
  *
- * Copyright 2000-2016 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2000-2018 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
Last time this file changed was in 2016

Reply:
But the script that generated it was changed in 2018. Therefore the derived file should change it's date. Yes this will get maybe a false positive, but it will also never miss anything.

Reply:
I don't quite agree with you there.  If the *content* in the derived file has changed, then I would agree.  Now, it's a year update just to update the year, that's kinda circular logic.

But you know...  meh, this is small crap, I'm not gonna fight you on this...